integration-test-apps/manual-instrumentation/flask/create_flask_app.py

```diff
@@ -7,16 +7,21 @@
 
 import boto3
 import requests
-from flask import Flask, request, session
+from flask import Flask, session
+import logging
 from opentelemetry import trace
 from opentelemetry.sdk.extension.aws.trace.propagation.aws_xray_format import (
     TRACE_ID_DELIMITER,
     TRACE_ID_FIRST_PART_LENGTH,
     TRACE_ID_VERSION,
 )
 
+if os.environ.get('SAMPLE_APP_LOG_LEVEL') == 'ERROR':
+    log = logging.getLogger('werkzeug')
```

Comment:
What is `werkzeug`?

Reply:
It is the logger used by the `flask` framework for its logs as I found out in [this SO post](https://stackoverflow.com/a/18379764/7460739). By "getting" that logger, we are modifying its log level (which will get very noisy when the Load Generator stress-tests the app).